Dockerfile

```diff
@@ -117,3 +117,5 @@ COPY hack/patches/* /tmp/install_deps/patches/
 COPY hack/apply_patches.sh /tmp/install_deps/
 # Apply patches here to be able to patch selinon as well
 RUN /tmp/install_deps/apply_patches.sh
+
+RUN pip3 uninstall -y protobuf && pip3 install packaging appdirs && pip3 install --upgrade --no-binary :all: protobuf==3.3.0
```

Comment:
We encountered a very weird Import Error with Google Cloud lib. Even if the lib is installed properly and we can see manually that it is available in import-path, but still the program throws Import Error for "protobuf" !

I discussed it in matter-most and Saket/Slavek shared the above work-around.
Reference: https://chat.openshift.io/rh-devtools/pl/5e6sbcsyz3fuxfw1zs4n97hwww